test/conformance/conformance_test.sh

```diff
@@ -25,5 +25,5 @@ if diff -u test/conformance/testdata/conformance.txt test/conformance/conformanc
   echo PASS
   exit 0
 fi
-echo 'See instructions in test/conformance/README.md'
+echo 'See instructions in test/conformance/README.md'l
```

Comment:
What's this line for?

Reply:
My God, I don't know also. I have reset it.